cmd/plugin/builder/template/plugintemplates/plugin-tooling.mk.tmpl

```diff
@@ -10,9 +10,9 @@ ifeq ($(GOHOSTOS),windows)
 endif
 
 # Build and version information
-PLUGIN_BUILD_SHA ?= $(shell git describe --match=$(git rev-parse --short HEAD) --always --dirty)
+PLUGIN_BUILD_SHA ?= $(shell git describe --match=$(git rev-parse --short HEAD) --exclude "test/e2e/framework/*" --always --dirty)
```

Comment:
Is this necessary? This would mean having some opinions of the repo tags in the author's repo.

Reply:
@vuil makes sense, reverted the change for plugin templates